### PR TITLE
Bugfix: include GA on all pages by domains #3568

### PR DESCRIPTION
**Pull request by amarynets**
Need test it on dev server

### CODE REVIEW

File: digitalglarus/templates/new_base_glarus.html [hunk 1]
```diff
@@ -46,16 +46,19 @@
...
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-      ga('create', 'UA-62285904-1', 'auto');
-      ga('send', 'pageview');
-
-    </script>
+    <!--<script src="//www.google-analytics.com/analytics.js" async></script><script>-->
```
Comment (pcoder) [inline digitalglarus/templates/new_base_glarus.html]: Can we remove it, if we are not using it?

File: hosting/templates/hosting/base.html [hunk 2]
```diff
@@ -45,16 +45,19 @@
...
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-  ga('create', 'UA-62285904-5', 'auto');
-  ga('send', 'pageview');
-
-</script>
+<!--<script>-->
```
Comment (pcoder) [inline hosting/templates/hosting/base.html]: Same here. Can we remove the commented out code?

File: hosting/templates/hosting/beta.html [hunk 3]
```diff
@@ -31,16 +31,19 @@
...
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-  ga('create', 'UA-62285904-5', 'auto');
-  ga('send', 'pageview');
-
-</script>
+<!--<script>-->
```
Comment (pcoder) [inline hosting/templates/hosting/beta.html]: Here too.

File: ungleich/templates/cms/ungleichch/base_ungleich.html [hunk 4]
```diff
@@ -60,16 +64,16 @@
     {% bootstrap_javascript %}
     {% endaddtoblock %}
     {% addtoblock "js" %}
-    <script>
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    <!--<script>-->
```
Comment (pcoder) [inline ungleich/templates/cms/ungleichch/base_ungleich.html]: Here too.

File: ungleich_page/templates/ungleich_page/landing.html [hunk 5]
```diff
@@ -32,25 +32,28 @@
...
-            a.src = g;
-            m.parentNode.insertBefore(a, m)
-        })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
-
-        ga('create', 'UA-62285904-1', 'auto');
-        ga('send', 'pageview');
-
-    </script>
+    <!--&lt;!&ndash; Google tracking &ndash;&gt;-->
+    <!--<script src="//www.google-analytics.com/analytics.js" async></script>-->
```
Comment (pcoder) [inline ungleich_page/templates/ungleich_page/landing.html]: Here too.

**Comment by amarynets** [comment]
@pcoder Yes, I will do it

**Pull request merged**